src/actions/operator.ts

```diff
@@ -261,25 +261,30 @@ export class YankOperator extends BaseOperator {
     if (end.isEarlierThan(start)) {
       [start, end] = [end, start];
     }
-    end = new Position(end.line, end.character + 1);
+    let extendedEnd = new Position(end.line, end.character + 1);
```

Comment:
Instead of modifying the `end` parameter we create a local variable.

Reply:
Err, sorry I meant just within the `if` block.

Reply:
I understand. 

The original code mutad the `end` parameter by adding 1 to its length, which is rather ugly.

I introduced a variable to hold the mutated parameter and used the original parameter in my code.

What's your opinion?